include/faker-cxx/music.h

```diff
@@ -15,7 +16,7 @@ namespace faker::music
  * faker::music::artist() // "Nirvana"
  * @endcode
  */
-FAKER_CXX_EXPORT std::string_view artist();
+FAKER_CXX_EXPORT std::string_view artist(Locale locale = Locale::en_US);
```

Comment:
Add doxygen comments about new param

Reply:
Do i need to add these comments manually?

Reply:
I have added comments. Can you please review it? Thank you